cool.py

```diff
@@ -141,23 +131,31 @@ def get_tile(url):
         dict: The 'results' dictionary of the response json (location, score, and matchAddress)
     """
 
-    response = requests.get(url, timeout=5)
+    session = _get_retry_session()
+
+    try:
+        response = session.get(url, timeout=5)
+        response.raise_for_status()  #: raise an exception if the status code is not 200 OK
```

Comment:
**thought**: you might need to be careful here as a 404 will throw but we probably just want to skip it from the mosaic unless all of the tiles 404 and you just have a white image.